sentry/templates/deployment-sentry-web.yaml

```diff
@@ -8,6 +8,10 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 spec:
+{{- if .Values.sentry.web.enableRecreate }}
+  strategy:
```

Comment:
I think we should call the value .Values.sentry.web.strategyType with a default value of RollingUpdate ?

Reply:
Done!